src/selectors.js

```diff
@@ -1,9 +1,15 @@
 import { matchPath } from "react-router"
+import isObject from 'lodash/isObject'
```

Comment:
`isObject` is quite simple, so, we can define `isObject` here and don't need to maintain external dependency. 
```
const isObject = (value) => (value != null && (typeof value === 'object' || typeof value === 'function'))